<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="quit">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>410</y>
      <width>92</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>751</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="layout_scene"/>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>551</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_damping">
       <property name="text">
        <string>damping : 0.04</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_coeff">
       <property name="text">
        <string>bounce coeff : 1.00</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSlider" name="bounce_coeff_slider">
       <property name="value">
        <number>99</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="damping_slider">
       <property name="value">
        <number>20</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="bounces_number">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_bounces">
       <property name="text">
        <string>bounces on wall :</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="draw">
       <property name="text">
        <string>Draw</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>quit</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>gridLayoutWidget</zorder>
   <zorder>draw</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>806</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
